docs-core/src/main/java/com/sismics/docs/core/dao/ReviewDao.java

```diff
@@ -0,0 +1,75 @@
+package com.sismics.docs.core.dao;
```

Comment:
You need to resolve all the merge conflicts with main. Run `git pull origin main` then manually resolve them on your end.